### PR TITLE
Tasks logic test

### PR DESCRIPTION
**Pull request by eugene-chr**

### CODE REVIEW

File: src/store/tasks/__tests__/logic.spec.js [hunk 1]
```diff
@@ -0,0 +1,778 @@
...
+  saveTaskSuccess,
+  setTaskDeadlineSuccess,
+  setTaskDoneSuccess,
+  setTaskOrderSuccess,
+  setTasks,
+  updateTaskSuccess,
+} from '../actions';
+import normalizeTasks from '../normalize';
+
+const fetchTasksLogic = tasksLogic[0];
```
Comment (plzen) [inline src/store/tasks/__tests__/logic.spec.js]: You can use named export for each logic

File: src/store/tasks/__tests__/logic.spec.js [hunk 2]
```diff
@@ -0,0 +1,778 @@
...
+      const data = {
+        id: 'taskID',
+      };
+      beforeEach(() => {
+        add.mockResolvedValue(data);
+        saveTaskLogic.process({ getState, action, db, firestore }, dispatch, done);
+      });
+
+      it('calls collection()', () => {
+        expect(collection.mock.calls[1][0]).toMatch('tasks');
```
Comment (plzen) [inline src/store/tasks/__tests__/logic.spec.js]: Use `toHaveBeenNthCalledWith`

Comment (eugene-chr) [reply src/store/tasks/__tests__/logic.spec.js]: done

File: src/store/tasks/__tests__/logic.spec.js (hunk 1 above)
Comment (eugene-chr) [reply src/store/tasks/__tests__/logic.spec.js]: done

**Pull request merged**